Sources/ActivityIndicators/Animations/ActivityIndicatorAnimationCirclePendulum.swift

```diff
@@ -0,0 +1,97 @@
+//
+//  ActivityIndicatorAnimationCirclePendulum.swift
+//  IBAnimatable
+//
+//  Created by phimage on 29/06/2018.
+//  Copyright © 2018 IBAnimatable. All rights reserved.
+//
+
+import Foundation
+
+import UIKit
+
+public class ActivityIndicatorAnimationCirclePendulum: ActivityIndicatorAnimating {
+
+  // MARK: Properties
+
+  fileprivate let duration: CFTimeInterval = 1
+  fileprivate let ratio: CGFloat = 7
+  fileprivate let ballCount: Int = 3
+
+  #if LG
```

Comment:
What means `LG`?

Reply:
Life's Good
_This activity indicator has been inspired from my new tv, to watch the world cup_

This override the color defined and set 3 colors.
Maybe `LG` could be used accidentally, If I could keep my eastern eggs, a longer compilation flag could be used

Reply:
Shall we have a property instead of a custom flag? Otherwise, the user needs to set the configuration in Podfile to enable it like `build_settings['OTHER_SWIFT_FLAGS'] = '-LG'`. If you think that's fun, shall we add it to ActivityIndicators.md . BTW: I have `LG` big TV at home too 📺

Reply:
ActivityIndicatorAnimation classes are not very accessible, but I could make `trueColor` as a `public var`
The compilation will change only the default value.

Yes I will add in doc

ps: same trick on triforce

Reply:
@phimage cool, thanks, LG(Life's Good), you make it even LB (Life's Better)😉